common/components/chrome/MainFooter.jsx

```diff
@@ -36,33 +36,44 @@ class MainFooter extends React.Component<{||}> {
       <React.Fragment>
       <div className="MainFooter-border"></div>
       <div className="MainFooter-footer container">
-        <div className="MainFooter-item col-xs-12 col-md-6">
+        <div className="MainFooter-item col-12 text-center">
           <h2>Sponsors Make It Possible</h2>
           <p>Support the acceleration of social change</p>
           <a className="btn btn-primary" href={url.section(Section.PartnerWithUs)}>
             PARTNER WITH US
           </a>
         </div>
-        {this._renderSponsors()}
+        <div className="MainFooter-sponsor-container col-12">
+            {/* {this._renderSponsors("Visionary")} */}
```

Comment:
Since the code is here let's leave the unused tiers uncommented so we can add them via config later.